src/library_browser.js

```diff
@@ -1268,7 +1268,7 @@ mergeInto(LibraryManager.library, {
       }
       var data = msg.data['data'];
       if (data) {
-        if (!data.byteLength) data = new Uint8Array(data);
+        data = new Uint8Array(data);
```

Comment:
this is data that just arrived in a message from a worker, so it is already a typed array - why copy it?